plugins/main/bridge/bridge_test.go

```diff
@@ -106,6 +107,8 @@ const (
 
 	ipamEndStr = `
     }`
+
+	ipamDatadir = "/var/lib/cni/networks/testConfig/"
```

Comment:
One thing you could do here avoid race issues is to create your own datadir for the test.  I've done an implementation of that in https://github.com/dcbw/plugins/commit/2ee31427f9156b57bfb6ec754fed419099eb9e9f .  Cherry-pick that commit from my fork into your PR and then pass `dataDir` to tc.createCmdArgs() and then *also* to countIPAMIPs().  That'll make sure nothing else messes with the datadir during the test.

Reply:
Thanks @dcbw for the suggestion, this solves the issue.
I updated the commits with this.